cabal-install/Distribution/Client/Setup.hs

```diff
@@ -1220,11 +1256,24 @@ installCommand = CommandUI {
      ++ " that, `configure` must be used.)\n"
      ++ " In contrast, without a sandbox, the flags to `install` are saved and"
      ++ " affect future commands such as `build` and `repl`. See the help for"
-     ++ " `configure` for a list of commands being affected.\n",
+     ++ " `configure` for a list of commands being affected.\n"
+     ++ "\n"
+     ++ "Installed executables will by default (and without a sandbox)"
+     ++ " be put into `~/.cabal/bin/`."
```

Comment:
firstly: i was not under the impression that this prevents a merge. please say if i am wrong.
secondly: beginners often report "i did cabal install cabal-install, but i still seem to be using the old version", and this note would address this misunderstanding. but perhaps i can improve the wording a bit, how about:

    Installed executables will by default (and without a sandbox) be put into the
    `bin` directory in the user's cabal directory (e.g. `$HOME/.cabal/bin` on linux).